includes/handle-redirects.php

```diff
@@ -32,10 +32,6 @@ function maybe_do_redirect() {
 
 	$path = untrailingslashit( str_replace( wp_parse_url( home_url(), PHP_URL_PATH ), '', sanitize_text_field( wp_unslash( $_SERVER['REQUEST_URI'] ) ) ) );
 
-	if ( ! empty( $_SERVER['QUERY_STRING'] ) ) {
```

Comment:
So we just ignore this? This seems like the wrong fix to me.

Reply:
See issue #15 - the query string is already present via `REQUEST_URI` so this code just adds it again. Why not just remove this?

Reply:
Can come back to this in the 1.0 milestone if you feel there's a more correct approach.